legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/testSuite/naming/naming.pure

```diff
@@ -129,7 +129,7 @@ function <<paramTest.BeforePackage>> meta::relational::tests::dbSpecificTests::s
                  '---\n'+
                 'default\n'+
                 'testTable3\n'+
-                'ABCDEFGHIJKLMNOPQRSTUVWXYZ,"0123456789","!\"#$%&()*+-./:;<=>?@[\]^_`{|}",abcdefghijklmnopqrstuvwxzy\n'+
+                'ABCDEFGHIJKLMNOPQRSTUVWXYZ,"0123456789","!\"\"#$%&()*+-./:;<=>?@[\]^_`{|}",abcdefghijklmnopqrstuvwxzy\n'+
```

Comment:
Since double-quote is part of data in this case, it should be enclosed in single quote
`'!\"#$%&()*+-./:;<=>?@[\]^_`{|}'`